src/main/java/uk/gov/companieshouse/api/accounts/util/ixbrl/accountsbuilder/AccountsBuilder.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.companieshouse.api.accounts.util.ixbrl.accountsbuilder;
+
+import java.io.IOException;
+
+/**
+ * Interface to call Accounts API to retrieve the accounts information.
+ *
+ * This functionality has not been implemented yet. Waiting for the API changes to be deployed;
+ * once, that happens the accounts builder implementation needs to be change.
+ **/
+public interface AccountsBuilder {
+    Object buildAccount() throws IOException;
```

Comment:
You could use generics here to prevent the need to cast.
```java
public interface AccountsBuilder<T> {
    T buildAccount();
}
```

public class SmallFullAccountsBuilderImpl implements AccountsBuilder<SmallFull> {
    public SmallFull buildAccount() {
    }
}